### PR TITLE
[PEER REVIEW] Revisions to basic authentication

### PR DESCRIPTION
**Pull request by tpoitras**
Docs Trello Card: https://trello.com/c/0Al6rvVt

@liggitt -- excellent job writing this stuff up. :) I didn't have to alter much, it was so detailed and well-explained. Can you just review my changes and make sure I didn't mess anything? Thanks. :)

### CODE REVIEW

File: admin_guide/service_accounts.adoc [hunk 1]
```diff
@@ -48,12 +49,12 @@ $ oc policy add-role-to-group edit system:serviceaccounts:managers -n top-secret
 
 == Enable service account authentication
 
-Service accounts authenticate to the API using tokens signed by a private RSA key.
+Service accounts authenticate to the API using tokens signed by a private RSA key. The authentication layer verifies the signature with a matching public RSA key.
```
Comment (liggitt) [inline admin_guide/service_accounts.adoc]: "verifies the signature *using* a matching public RSA key"?

Comment (liggitt) [inline admin_guide/service_accounts.adoc]: heh, you just moved it... I guess I'm copy editing myself, then

**Comment by liggitt** [comment]
LGTM

**Comment by adellape** [comment]
@tpoitras Needs a rebase.

**Comment by tpoitras** [comment]
Updated and rebased, @adellape :+1: :hamburger:

**Pull request closed without merging**